render/js/help.js

```diff
@@ -1,32 +1,43 @@
-const dataContainerRef = document.getElementById('data-container'); // A reference to the div with id "data-container"
-const searchFormRef = document.getElementById("search-form"); // A reference to the form with id "search-form"
 
-
-searchFormRef.addEventListener("submit", async (event) => {
-    event.preventDefault();
-    let searchInput = document.getElementById("search-box").value;
-    searchInput = parseInt(searchInput);
-    if (isNaN(searchInput)) {
-        dataContainerRef.innerHTML = "<h2>Invalid input</h2>";
-        return;
+function displayNothing(){
+    console.log("ran")
+    let arr = [
```

Comment:
Place these all into the same HTML class, so you can do query Selector all rather than grabbing each ID.